dist/index.html

```diff
@@ -3,9 +3,21 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>TheMealDB</title>
 <script defer src="main.bundle.js"></script></head>
 <body>
-    <h1>Hello webpack!</h1>
+    <header>
+        <h1>TheMealDB</h1>
+        <button class="icon-bar-button">
+            <span></span>
+            <span></span>
+            <span></span>
+        </button>
```

Comment:
- Here is what I am talking about 😊 You forgot to add the functionality of the add event listener click 👎  when the user opens the project from the mobile screen the bar will appear as you did 👌 , but we want when the user clicks at the bars the Navliks will appear to him 💯 .